ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -102,3 +102,4 @@ sunbird_bot_service_URL=/chatapi/bot
 sunbird_p1_reCaptcha_enabled={{sunbird_p1_reCaptcha_enabled|default('true')}}
 sunbird_p2_reCaptcha_enabled={{sunbird_p2_reCaptcha_enabled|default('true')}}
 sunbird_p3_reCaptcha_enabled={{sunbird_p3_reCaptcha_enabled|default('true')}}
+sunbird_phraseApp_token={{sunbird_phraseApp_token|''}}
```

Comment:
This is incorrect. Please update with default keywork

Reply:
Fixed

Reply:
what will happen if the variable is empty ? @Nik720

Reply:
If this token is empty then it will call the old method.

Reply:
Resolved